### ISSUE TITLE
Add Ability to Pan Canvas in View Mode

### ISSUE THREAD

**Issue by tristanraab** [CSCI-435-SE/excalidraw#8]
### Summary 
Currently there is no way to pan the canvas without a mouse. For accessibility, users should be able to use their keyboard to pan the canvas, potentially with the arrow keys.

### User Story
As a user, I want to be able to move the canvas around just using my keyboard.

### Rough Scope
Small

### Source
Project issue tracker
